File: InfoLoom/UI/src/mods/domain/StorageCompanyInfo.ts
import { Entity } from 'cs2/utils';
import { LocalizedString } from 'cs2/l10n';

export interface StorageCompanyInfo {
  EntityId: Entity;
  Brand: String;
  TransferRequests: number;
  Trips: number;
  OwnedVehicles: number;
  GuestVehicles: number;
}

File: InfoLoom/Systems/IndustrialSystems/StorageCompanies/Data/StorageCompanyInfo.cs
using System;
using Unity.Collections;
using Unity.Entities;

namespace InfoLoomTwo.Systems.IndustrialSystems.StorageCompanies.Data
{
    public struct StorageCompanyInfo 
    {
        public Entity EntityId;
        public Entity Brand;
        public int TransferRequests;
        public int Trips;
        public int OwnedVehicles;
        public int GuestVehicles;
    }
    public struct StorageCompanyUI
    {
        public Entity EntityId;
        public string Brand;
        public int TransferRequests;
        public int Trips;
        public int OwnedVehicles;
        public int GuestVehicles;
    }
}
